packages/@aws-cdk/aws-iotevents/lib/input.ts

```diff
@@ -8,11 +9,55 @@ import { CfnInput } from './iotevents.generated';
 export interface IInput extends IResource {
   /**
    * The name of the input
+   *
    * @attribute
    */
   readonly inputName: string;
+
+  /**
+   * The ARN of the input
```

Comment:
```suggestion
   * The ARN of the input.
```

Reply:
I've fix all same case in aws-iotevents 👍🏻